src/components/side-panel/layer-panel/color-range-selector.tsx

```diff
@@ -202,7 +230,7 @@ export const PaletteConfig = ({label, value, config: {type, options}, onChange})
     {type === 'select' && (
       <div className="color-palette__config__select">
         <ItemSelector
-          selectedItems={value}
+          selectedItems={[value]}
```

Comment:
shouldn't need to convert to array `ItemSelector` props.selectedItems can be a single object

Reply:
fixed